package.json

```diff
@@ -35,18 +35,17 @@
   "homepage": "https://github.com/libp2p/js-libp2p-mdns",
   "devDependencies": {
     "aegir": "^25.0.0",
-    "chai": "^4.2.0",
```

Comment:
Dep check is failing with chai removed, there must be some remaining usages of it.